syntax = "proto3";
package payload.manager.v1;

import "payload/manager/v1/stats.proto";

/*
  Operational inspection API.
*/
service PayloadAdminService {
  rpc Stats(StatsRequest) returns (StatsResponse);
}
